docker-compose.development.override.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+
+services:
+  ms: &ms
+    build: .
+    environment:
+      DATABASE_URL: postgres://postgres@db:5432/msf_dev
+
+    volumes:
+      - .:/usr/src/app
```

Comment:
Right, currently they do exist outside of the docker container, but inside the project root, but not the users `$HOME`. I did this so I could use an `msfconsole.rc` to assist in getting the env setup.

I will change the docker-compose command to include `-r`, that way we can mount `$HOME/.msf4/`